bindings_node/src/client.rs

```diff
@@ -129,6 +129,8 @@ pub async fn create_client(
   history_sync_url: Option<String>,
   log_options: Option<LogOptions>,
 ) -> Result<Client> {
+  let account_address = account_address.to_lowercase();
```

Comment:
This will lowercase the account_address being stored on line 183. (unchanged in this PR, so I can't comment on it.)

Reply:
curious why this is necessary